autoendpoint/src/routers/apns/error.rs

```diff
@@ -86,3 +87,24 @@ impl From<ApnsError> for ApiErrorKind {
         ApiErrorKind::Router(RouterError::Apns(e))
     }
 }
+
+impl ReportableError for ApnsError {
+    fn is_sentry_event(&self) -> bool {
+        matches!(self, ApnsError::ApnsUpstream(_))
```

Comment:
Following up to https://github.com/mozilla-services/autopush-rs/pull/632#discussion_r1601796508

> Yeah, looks like a merge error, but I'm going to include `ApnsUpstream` as a valid Sentry error. We shouldn't get many of them, and if we do, it would impact our delivery capacity for APNS, so we should at least record the error.

`ApnsUpstream` was already going to sentry, everything but `RouterError::Apns(ApnsError::SizeLimit(_)) | RouterError::Apns(ApnsError::Unregistered)` did in the previous code --  so it sounds like the intention is to be the same as the previous check (which goes with the current `metric_label` impl).

```suggestion
        !matches!(self, ApnsError::SizeLimit(_) | ApnsError::Unregistered)
```